.gitlab-ci.yml

```diff
@@ -122,7 +122,8 @@ deploy_latest_tag_to_docker_registries:
   rules:
     - if: '$POPULATE_CACHE'
       when: never
-    - if: '$CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+\.[0-9]+(-prerelease)?$/'
+    # Don't deploy latest tag if prerelease
+    - if: '$CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+\.[0-9]+$/'
```

Comment:
We also don't want this if we publish a 2.x release _after_ 3.x it out, right? Do we need a check for this later?

Reply:
Yep, that's handled in this subsequent PR, which we'll merge into the `release/v2` branch after we've cut that (and merged the v3 branch)
- https://github.com/DataDog/dd-trace-dotnet/pull/5357